app/main/views/suppliers.py

```diff
@@ -148,50 +146,31 @@ def supplier_update(id=None, step=None, substep=None):
         abort(e.status_code)
 
     if feature.is_active('SELLER_UPDATE'):
-        # add business/authorized representative contact details
-        if len(supplier['contacts']) > 0:
-            supplier['email'] = supplier['contacts'][0]['email']
-            supplier['phone'] = supplier['contacts'][0]['phone']
-            supplier['representative'] = supplier['contacts'][0]['name']
-        if len(supplier['address']) > 0:
-            supplier['address']['address_line'] = supplier['address']['addressLine']
-            supplier['address']['postal_code'] = supplier['address']['postalCode']
-        props = {"application": supplier}
-        props['basename'] = url_for('.supplier_update', id=current_user.supplier_code)
-        props['options'] = {'seller_update': feature.is_active('SELLER_UPDATE')}
-        props['form_options'] = {
-            'action': url_for('.supplier_update_save', id=current_user.supplier_code),
-            'submit_url':  url_for('.supplier_update_save', id=current_user.supplier_code),
-        }
-        rendered_component = render_component('bundles/SellerRegistration/ApplicantSignupWidget.js', props)
-
-        return render_template(
-            '_react.html',
-            component=rendered_component
-        )
-
-
-@main.route('/update/<int:id>', methods=['POST'])
-@main.route('/update/<int:id>/<path:step>', methods=['POST'])
-@main.route('/update/<int:id>/<path:step>/<path:substep>', methods=['POST'])
-@login_required
-def supplier_update_save(id=None, step=None, substep=None):
-    json = request.content_type == 'application/json'
-    form_data = from_response(request)
-    supplier_updates = form_data['application'] if json else form_data
-
-    try:
-        result = data_api_client.update_supplier(
-            current_user.supplier_code,
-            supplier_updates,
-            user=current_user.email_address
-        )
-    except APIError as e:
-        current_app.logger.error(e)
-        abort(e.status_code)
 
-    if json:
-        return jsonify(result)
+        if 'application_id' not in supplier:
+            # add business/authorized representative contact details
+            if len(supplier.get('contacts', [])) > 0:
+                supplier['email'] = supplier['contacts'][0].get('email')
+                supplier['phone'] = supplier['contacts'][0].get('phone')
+                supplier['representative'] = supplier['contacts'][0].get('name')
+            if len(supplier.get('address', [])) > 0:
+                supplier['address']['address_line'] = supplier['address']['addressLine']
+                supplier['address']['postal_code'] = supplier['address']['postalCode']
+            del supplier['id']
+            supplier['status'] = 'saved'
+            application = data_api_client.create_application(current_user.id, supplier)
```

Comment:
will need to rework this with my changes on the other branch - the application will need to bet set on the user